vue-thacer/src/assets/js/thacer-map-create-layer.js

```diff
@@ -64,71 +64,80 @@ export function setCeramLayer(ceramLayer) {
 }
 
 export function createFeatureLayerSecteurs(ceram, markerClusterGroupCeram, map) {
-  /* global L */
-  return L.mapbox
-    .featureLayer()
-    .loadURL(import.meta.env.VITE_API_URL + 'geojson/secteurs.geojson')
-    .on('ready', function (e) {
-      // function select ceram according to the secteurs ID on the clicked secteur feature
-      e.target.eachLayer(function (layer) {
-        layer.eachLayer(function (e) {
-          let string = ''
-          let biblio = ''
-          if (
-            !(
-              // GTh means "Guide de Thasos"
-              (
-                (layer.feature.properties.GTh == '') |
-                (layer.feature.properties.GTh == 'null') |
-                (layer.feature.properties.GTh == undefined)
-              )
-            )
-          ) {
-            string = ' GTh' + layer.feature.properties.GTh
-          }
-          if (
-            !(
-              (layer.feature.properties.Référenc == '') |
-              (layer.feature.properties.Référenc == 'null') |
-              (layer.feature.properties.Référenc == undefined)
+  // create a new leaflet GeoJSON layer
+  const layer = L.geoJSON();
```

Comment:
have got a "not defined" by eslint. Was it a Mapbox? should it be define somewhere?

Reply:
@chtzv @AlZorba the `/* global L *`/ is an instruction to eslint (to tell him that the variable exist). It must be kept